features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/trace/FeatureSdkCoreExtTest.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.trace
+
+import com.datadog.android.api.feature.Feature
+import com.datadog.android.api.feature.FeatureSdkCore
+import com.datadog.android.utils.forge.Configurator
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.kotlin.argumentCaptor
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.verify
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(Configurator::class)
+internal class FeatureSdkCoreExtTest {
+
+    @Mock
+    lateinit var mockSdkCore: FeatureSdkCore
+
+    @StringForgery
+    lateinit var fakeTraceId: String
+
+    @StringForgery
+    lateinit var fakeSpanId: String
+
+    @Suppress("UNCHECKED_CAST")
+    @Test
+    fun `M add the active trace to trace context W addActiveTraceToContext() `() {
+        // Given
+        val expectedThreadName = Thread.currentThread().name
+        val activeTraceContextName = "context@$expectedThreadName"
+
+        // When
+        mockSdkCore.addActiveTraceToContext(fakeTraceId, fakeSpanId)
+
+        // Then
+        val traceContext: MutableMap<String, Any?> = mutableMapOf()
+        argumentCaptor<(MutableMap<String, Any?>) -> Unit> {
+            verify(mockSdkCore).updateFeatureContext(eq(Feature.TRACING_FEATURE_NAME), capture())
+            firstValue.invoke(traceContext)
```

Comment:
`traceContext` can be directly in lambda

```suggestion
        argumentCaptor<(MutableMap<String, Any?>) -> Unit> {
            verify(mockSdkCore).updateFeatureContext(eq(Feature.TRACING_FEATURE_NAME), capture())
            val traceContext: MutableMap<String, Any?> = mutableMapOf()
            firstValue.invoke(traceContext)
```